tests/testthat/_snaps/format_integer64.md

```diff
@@ -98,66 +98,66 @@
       pillar(x, width = 16)
     Output
       <pillar>
-      <nanotime>      
+      <int64>         
```

Comment:
This is wrong. Where does it come from?

Reply:
From `{bit64}`:

``` r
library(pillar)
suppressPackageStartupMessages(library(bit64))

x <- as.integer64(c(0, 1e11, 1e12) + 0:2)
pillar(x)
#> <pillar>
#>       <int64>
#>             0
#>  100000000001
#> 1000000000002
```

<sup>Created on 2022-10-16 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>